Framework/src/QcInfoLogger.cxx

```diff
@@ -50,11 +50,31 @@ void QcInfoLogger::setDetector(const std::string& detector)
   ILOG(Debug, Support) << "Detector set to " << detector << ENDM;
 }
 
+void QcInfoLogger::setRunPartition(int run, std::string& partitionName)
```

Comment:
Is there any reason to always set those two in one method, but others separately?